### PR TITLE
Add a page for external standards catalogues

### PR DESCRIPTION
**Pull request by nahteb**
New page for external catalogues:  

Screenshot 2021-11-04 at 14 00 19  

Link from the standards page:    

Screenshot 2021-11-04 at 14 00 33

### CODE REVIEW

**Review by charlesbaird-dsa — APPROVED** [review]
I like it. Simple but effective...

File: content/standards/external-standards.html.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: External Standards Catalogues
+
+---
+
```
Comment (jamietanna) [inline content/standards/external-standards.html.md:5]:
```suggestion
---
```

File: content/standards/index.html.md.erb [hunk 2]
```diff
@@ -33,5 +33,8 @@ The Data Standards Catalogue currently contains the following Standards:
   </table>
 </div>
 
+## External Standards Catalogues
+You can find a list of external standards catalogues [here](/standards/external-standards.html).
+
```
Comment (jamietanna) [inline content/standards/index.html.md.erb:38]:
```suggestion
## External Standards Catalogues

You can find a list of external standards catalogues [here](/standards/external-standards.html).
```

File: content/standards/index.html.md.erb [hunk 3]
```diff
@@ -33,5 +33,8 @@ The Data Standards Catalogue currently contains the following Standards:
   </table>
 </div>
 
+## External Standards Catalogues
+You can find a list of external standards catalogues [here](/standards/external-standards.html).
```
Comment (jamietanna) [inline content/standards/index.html.md.erb:37]: :question: I've read before (sorry I can't remember where!) that `here` isn't the most accessible for screen readers - could we reword the sentence so the link can make more sense?

File: content/standards/external-standards.html.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+# External Standards Catalogues
+
+- [Criminal Justice System Exchange Data Standards Catalogue](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/360582/cjs-data-standards-catalogue-5.0.pdf)
```
Comment (jamietanna) [inline content/standards/external-standards.html.md:8]: Is there only a PDF, no other versions?

**Review by jamietanna — CHANGES_REQUESTED** [review]
Few comments, but looks good - are we thinking that Markdown is easier for contributors compared to i.e. CSV?

File: content/standards/external-standards.html.md (hunk 4 above)
Comment (nahteb) [reply content/standards/external-standards.html.md:8]: I can't see another one :( there is this page but it just links back to the PDF which has the actual content https://www.gov.uk/guidance/criminal-justice-system-data-standards-forum-guidance#data-standards-catalogue

Comment (charlesbaird-dsa) [reply content/standards/external-standards.html.md:6]: I've only seen the PDF...

File: content/standards/external-standards.html.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+title: External Standards Catalogues
+---
+# External Standards Catalogues
+
+- [Criminal Justice System Exchange Data Standards Catalogue](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/360582/cjs-data-standards-catalogue-5.0.pdf)
+- [NHS information standards](https://digital.nhs.uk/data-and-information/information-standards)
+- [ONS data standards](https://www.ons.gov.uk/aboutus/transparencyandgovernance/datastrategy/datastandards)
```
Comment (charlesbaird-dsa) [inline content/standards/external-standards.html.md:8]:
```suggestion
- [ONS data standards](https://www.ons.gov.uk/aboutus/transparencyandgovernance/datastrategy/datastandards)
- [LGA data standards](https://standards.esd.org.uk/)
```

**Review by charlesbaird-dsa — COMMENTED** [review]
Suggested the LGA standards catalogue link as per Em's issue.

File: content/standards/index.html.md.erb [hunk 6]
```diff
@@ -33,5 +33,9 @@ The Data Standards Catalogue currently contains the following Standards:
   </table>
 </div>
 
+## External Standards Catalogues
+
+We also have a list of [external standards catalogues](/standards/external-standards.html).
+
```
Comment (jamietanna) [inline content/standards/index.html.md.erb:39]:
```suggestion
```

File: content/standards/index.html.md.erb [hunk 7]
```diff
@@ -21,7 +21,7 @@ The Data Standards Catalogue currently contains the following Standards:
     </thead>
     <tbody class="govuk-table__body">
       <% current_resource.children.sort_by{ |page| get_link_text(page).downcase }.each do |f| %>
-        <% if f.content_type.include? "html" %>
+        <% if f.path.include? "index.html" %>
```
Comment (jamietanna) [inline content/standards/index.html.md.erb:24]: Alternatively, we could add something to the front matter to say `exclude_from_list`?

Comment (nahteb) [reply content/standards/index.html.md.erb]: good point - have tried your suggestion, is this what you meant? https://github.com/alphagov/data-standards-authority/pull/100/commits/128e3a5927a90720bed76290b684cbf626027024

File: content/standards/index.html.md.erb (hunk 3 above)
Comment (nahteb) [reply content/standards/index.html.md.erb]: is this ok? https://github.com/alphagov/data-standards-authority/pull/100/commits/bdaabd3c0a6bcae072b1e48e47ab7297338f13ab

**Review by jamietanna — APPROVED** [review]

**Comment by jamietanna** [comment]
Checked locally and looks good to me :rocket:

**Pull request merged**